autoload/codefmt.vim

```diff
@@ -252,6 +262,23 @@ function! codefmt#GetSupportedFormatters(ArgLead, CmdLine, CursorPos) abort
   return join(l:groups[0] + l:groups[1] + l:groups[2], "\n")
 endfunction
 
+""
+" Returns whether there is a default formatter available for the current
```

Comment:
Could you make this `@public` and regenerate the vimdoc?

It should have a sample of how to use it somewhere in the help docs, but for that part I'm happy to merge it w/o and add the docs after.

Reply:
Done (two years later :)